src/scss/dp/overrides/components/_coverage.scss

```diff
@@ -0,0 +1,29 @@
+.coverage {
+  &-search {
+    &__results {
+      display: flex;
+      justify-content: space-between;
+      align-content: center;
+      align-items: baseline;
+      gap: .5em;
+      padding: 0.5em 0;
+      margin-bottom: 0;
+      &:first-child {
+        border-top-width: 2px !important;
+      }
+
+      & .ons-btn__inner {
```

Comment:
Is the `&` here on its own (not part of building a string) only repeating the heirarchy `.ons-btn__inner` is defined within?

Reply:
The intention is to build this
```css
<element class="coverage-search__results">
  …
    <element class="ons-btn__inner">
```

Reply:
Wouldn't the nesting (the hierarchy) take care of that? As in 

```
foo {
  bar {
    color: black
  }
}
```

becomes

```
foo bar {
  color: black
}
```

Reply:
Yep, my bad for leaving it in. Will push an update